views/components/home-tab.pug

```diff
@@ -1,12 +1,13 @@
 div(id='home', class='col s12')
+  
   img(src='/images/officebeatz-banner.png', class='banner')
-  div(class='title-container z-depth-1')
-    a(class='songDisplay') Current song: 
-    a(id='titleID', class='songDisplay')
-    a(class='songDisplay')  by 
-    a(id='artistID', class='songDisplay')
+  //- div(class='title-container z-depth-1')
+  //-   a(class='songDisplay') Current song: 
+  //-   a(id='titleID', class='songDisplay')
+  //-   a(class='songDisplay')  by 
+  //-   a(id='artistID', class='songDisplay')
   div(class='content-container')
-    p(id='stuff') Welcome to <span id='accented'>#{title}</span>, a simple web application designed to get you out of your seat and on your feet. Prolonged sitting even for as little as one hour, damages the health of our blood vessels, and over time increases our risk of cardiovascular disease. Standing, stretching, or taking a short walk every 30-60 minutes can help to lessen the damaging effects of sitting.
+    p(id='stuff') Welcome to <span id='accented'>#{title}</span>, a simple web application designed to get you out of your seat and on your feet. Prolonged sitting even for as little as one hour, damages the health of our blood vessels, and over time increases our risk of cardiovascular disease. Standing, stretching, or taking a short walk every 15-45 minutes can help to lessen the damaging effects of sitting.
     br
     p(id='stuff') <span id='accented'>#{title}</span> will help improve your health by reminding you to get up and move through the power of your favorite songs. Unlike boring or annoying alarm reminders, OfficeBeatz, we hope, will facilitate a fun, energizing environment in your workplace, all while improving your health!
```

Comment:
The ID shouldn't be `stuff` and there shouldn't be two elements with the same ID. Same with `accented`

Reply:
that's from the last group, i'll make a ticket for now